### DISCUSSION TITLE
Pimcore Academy - Possible ways for attaching additional information to relations

### DISCUSSION THREAD

**Discussion by abdellahrk** [pimcore/pimcore#16724] [Q&A / Support] [answered]
Hello, I have completed the Pimcore Senior Level but for the final exam, I cannot pass the question `What are possible ways for attaching additional information to relations?` In the video and the documentation, the correct answer has to be `Use advanced many-to-many relation type.` But this does not allow me to pass. 
I even selected all the answers except `This requirement cannot be modelled with Pimcore.` and some other combinations but not possible.

**Comment by patriciomg** [comment, marked as answer]
Hi Abdel,
this should help you :) 
-  Use block with many-to-one relation and additional data attributes.
-  Use advanced many-to-many relation type.
-  Use a field collection with many-to-one relation and additional data attributes.

link:  https://pimcore.com/docs/platform/Pimcore/Objects/Object_Classes/Data_Types/Relation_Types/

regards!


edit: added link.

**Reply by abdellahrk** [reply]
Hi @patriciomg, thank you very much for unlocking me. 
And thank you for the link. I have been unlocked. 
Thank you.

**Reply by patriciomg** [reply]
don't mention it :) 
btw If someone else get locked too, don't forget to mark this question as answered.

have a nice day!
